tests/test_013_surface_water.py

```diff
@@ -66,19 +65,18 @@ def test_gdf_lake():
     ds["inflow"] = (("time",), [100.0])
     gdf_lake = gpd.GeoDataFrame(
         {
-            "name": ["0", "0", "1"],
-            "lakeno": [0, 0, 1],
+            "name": ["lake_0", "lake_0", "lake_1"],
             "strt": [1.0, 1.0, 2.0],
             "clake": [10.0, 10.0, 10.0],
             "INFLOW": ["inflow", "inflow", None],
```

Comment:
all caps?

Reply:
Yes, right now these options (like INFLOW) need to be all caps in nlmod. This is probably because in the manual of MODFLOW 6 these settings are also displayed in caps (I do not think MODFLOW 6 cares if it is in caps or not):

![image](https://github.com/user-attachments/assets/5725f092-03f8-444d-8e50-24bb49b04273)